Tools/config/ax_boost_base.m4

```diff
@@ -92,8 +92,11 @@ if test "x$want_boost" = "xyes"; then
     libsubdirs="lib"
     ax_arch=`uname -m`
     case $ax_arch in
-      x86_64|ppc64|s390x|sparc64|aarch64)
-        libsubdirs="lib64 lib lib64"
+      x86_64)
+        libsubdirs="lib64 libx32 lib lib64"
+        ;;
+      ppc64|s390x|sparc64|aarch64|ppc64le)
+        libsubdirs="lib64 lib lib64 ppc64le"
```

Comment:
That addition of `ppc64le` to `libsubdirs` looks rather suspect...

(And unrelated to your change, why's `lib64` in there twice?)